### PR TITLE
Add nginx integration

### PR DESCRIPTION
**Pull request by dzmitry-rudnouski**

### CODE REVIEW

File: defaults/main.yml [hunk 1]
```diff
@@ -42,3 +42,7 @@ sonar_db:
 sonar_store: "https://sonarsource.bintray.com/Distribution/sonarqube"
 # local download path
 download_path: "/tmp/"
+
+sonar_proxy:
```
Comment (arno49) [inline defaults/main.yml]: In my opinion role should provide splitted proxy feature by protocol:
- http
- https

And user should be able to choose  exact protocols he want.

File: molecule/default/molecule.yml [hunk 2]
```diff
@@ -7,7 +7,12 @@ lint:
   name: yamllint
 platforms:
   - name: instance
-    image: centos:7
+    image: solita/ubuntu-systemd:latest
```
Comment (arno49) [inline molecule/default/molecule.yml]: please add another molecule role for this case case

File: tasks/system/Linux.yml [hunk 3]
```diff
@@ -30,7 +30,6 @@
       state: "directory"
       owner: "{{ sonar_user }}"
       group: "{{ sonar_group }}"
-      mode: "0555"
```
Comment (arno49) [inline tasks/system/Linux.yml]: why this mode changed while installation ?

File: tasks/system/Linux.yml [hunk 4]
```diff
@@ -83,9 +83,16 @@
     systemd:
       daemon_reload: True
 
-  - name: "Start SonarQube server"
+  - name: "Enable sonarqube.service"
     service:
       name: "sonarqube"
       enabled: True
 
+  - name: "Render nginx config"
```
Comment (arno49) [inline tasks/system/Linux.yml]: split http/https servers and define in name that it's nginx configs

File: tasks/system/Linux.yml (hunk 3 above)
Comment (dzmitry-rudnouski) [reply tasks/system/Linux.yml]: it's not changed, but sonar user can't unarchive zip into /opt/sonarqube dir that have dr-xr-xr-x mode

File: tasks/system/Linux.yml [hunk 5]
```diff
@@ -51,11 +50,12 @@
     register: sonar_artifact
 
   - name: "Install SonarQube {{ sonar_major_version }}.{{ sonar_minor_version }}"
+    become_user: "{{ sonar_user }}"
```
Comment (arno49) [inline tasks/system/Linux.yml]: move down, pls

File: templates/nginx.http.conf.j2 [hunk 6]
```diff
@@ -0,0 +1,18 @@
+upstream sonar {
```
Comment (arno49) [inline templates/nginx.http.conf.j2]: combine into single template

File: tasks/system/Linux.yml (hunk 4 above)
Comment (dzmitry-rudnouski) [reply tasks/system/Linux.yml]: Done

**Comment by arno49** [comment]
Nice work, thanks a lot !

**Pull request merged**